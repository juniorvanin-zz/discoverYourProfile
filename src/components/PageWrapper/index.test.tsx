import React from "react";
import { render } from "@testing-library/react";

import { PageWrapper } from "./";

jest.mock("components", () => {
  return {
    Header: ({ title }: { title: string }) => <h1> title</h1>
  };
});

describe("PageWrapper", () => {
  it("creates page wrapper using title and children", () => {
    const { container } = render(
      <PageWrapper title="some title" children={<p>some children</p>} />
    );

    expect(container).toMatchSnapshot();
  });
});
