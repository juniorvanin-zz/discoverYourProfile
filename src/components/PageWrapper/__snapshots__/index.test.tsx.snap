// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageWrapper creates page wrapper using title and children 1`] = `
<div>
  <main>
    <h1>
       
      some title
    </h1>
    <section
      class="main-content"
    >
      <p>
        some children
      </p>
    </section>
  </main>
</div>
`;
